### ISSUE TITLE
Guild Vendors "Buy window" Error Tone or Selling nothing. 

### ISSUE THREAD

**Issue by Rigginit** [DarkstarProject/darkstar#4395]
**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [] checked the commit log to see if my issue has been resolved since my server was last updated
^Fresh install less than 12 hours ago

**_Client Version_** (type `/ver` in game) **.**   30171103_1

**_Server Version_** (type `!revision` in game) **:** f643501963854f1b763aec94c9ee730d1812b1c7

**_Source Branch_** (master/stable) **:** master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

Try to buy something from Guild vendors and "Buy" button gives error tone or a blank sales window. Not all vendors, and I could only narrow it down to find the problematic NPCS have the 4 digit guild ID in the script/zone/****/NPC.  Not all 4 digit NPCs have errors (Cooking Guild NPCs work for example) 

NPCs Ive found (not a complete list, just the few Ive checked):
Northern San d'Oria    - Woodworking Guild    - Chaupire
Windurst Woods         - Clothcraft Guild            -Kuzah Hpirohpon
Windurst Woods         - Bonecraft Guild            -Retto_Marutto

**Comment by teschnei** [comment]
Is it any guild ID over 255, perhaps?  Maybe it was broken by one of the
commits to remove warnings

Is there any message in the log?  We have a debug message for not being
able to find a guild ID

**Comment by Rigginit** [comment]
Sorry, I'm still new. if by the log you mean DSGame-Server window then no error is shown.  
The scripts for the NPCs are where I'm seeing the ID. I may have called it by the wrong name earlier but its this line:
if (player:sendGuild(5142,8,23,3)) then ...  
the working NPC for the guild has "514" and the other "5142" again not even sure this is where the problem lies.

**Comment by teschnei** [comment]
Yep, dsgame window is the log (also the log file which is the same thing)

If 514 works then my guess was wrong, guess we'll have to look into it more
